lighthouse-core/test/report/html/renderer/report-renderer-test.js

```diff
@@ -115,15 +115,18 @@ describe('ReportRenderer', () => {
       const container = renderer._dom._document.body;
       const output = renderer.renderReport(sampleResultsCopy, container);
 
+      function isPWAGauge(el) {
```

Comment:
pwa category is failing for the sample report json now, which means the pwa__wrapper class isn't set. Changed the test to look at text content instead.

Reply:
ah well I had to update the sample artifacts anyways, but I'm gonna keep this test change.